### DISCUSSION TITLE
Invalid URL: Protocol must be one of: https:, mailto:.

### DISCUSSION THREAD

**Discussion by Janaka-Steph** [MetaMask/snaps#2321] [Q&A]
I just installed the snap template with npx @metamask/create-snap my-snap.

Running the demo app, clicking "Send Message" fails with:
`An error happened: Invalid URL: Protocol must be one of: https:, mailto:.`

**Comment by czepluch** [comment]
Seeing the same. Could you post the solution @Janaka-Steph, since you closed the issue?

**Comment by czepluch** [comment]
Ah, dublicate it seem. Solution is here: https://github.com/MetaMask/snaps/issues/2324

**Reply by Janaka-Steph** [reply]
Yes, see the issue :)
